imports/plugins/included/payments-paypal/server/startup/startup.js

```diff
@@ -0,0 +1,9 @@
+import { Reaction, Hooks } from "/server/api";
+
+Hooks.Events.add("afterCoreInit", () => {
+  Reaction.addRolesToDefaultRoleSet({
```

Comment:
Shouldn't this permission be defined on the routes that are defined in the `register.js`?